uvm/js/common/ungrid/GridFilterController.js

```diff
@@ -133,10 +133,13 @@ Ext.define('Ung.cmp.GridFilterController', {
             var str = [], filtered = false;
 
             Ext.Array.each(cols, function (col) {
```

Comment:
No to all of this.

For one, it's an exception for tags.

For another, it's matching on the column identifier.

Look instead to use a field definition for conversions like we do in uvm/js/common/util/_Map.js.  the fields definition already has a tag definition.  Look into using the convert field there as is used in other fields.   You may need to properly associate the tags field for Sessions, Devices, Users, Hosts.

Reply:
What I'm saying is look into a more general way to perform the getFilters on the tags fields.

For example, the current code looking at TableConfig to get a renderer function defined for the column.  That seems like a more general approach than matching on the tag name itself.